instance/static/html/instance/index.html

```diff
@@ -0,0 +1,77 @@
+<top-bar>
+  <ul class="title-area">
+    <li class="name">
+      <h1><a href="#">OpenCraft - Instances Manager</a></h1>
```

Comment:
I personally find it confusing that OpenCraft is the name of this instance manager software as well as the company. I would suggest renaming it to "OpenCraft IM" (Instance Manager) or something ?

Also it would be cool to put the OpenCraft logo in the Readme or in a footer as "powered by OpenCraft".

Oh, BTW, I think "Instance**s** Manager" is not wrong but it's not the usual way this would be said in English - when describing a type of manager, the word before is not usually pluralized - e.g. "Project Manager", "Program Manager", "Tool Manager", "Process Manager", or (in this case) "Instance Manager".